### PR TITLE
key user vs developer ui dev + custom vs. generated ui

### PR DESCRIPTION
**Pull request by Liebherr**
Hi Arianna or Achim, can you please review.
I did on QA extensively already, but perhaps another two eyes still find something obvious.
Thanks and Regards
Ulrike

### CODE REVIEW

File: tutorials/abap-custom-ui-bas-develop-s4hc/abap-custom-ui-bas-develop-s4hc.md [hunk 1]
```diff
@@ -22,10 +22,10 @@ author_profile: https://github.com/Liebherr
...
   - This process of custom UI development is the way to go if the UI generated within the SAP S/4HANA Cloud system does not match your needs
 
-## Intro
-You can get a generated UI for your custom business object and adapt it to your needs within the SAP S/4HANA Cloud system (in-app-extensibility). As building UIs in SAP BTP (side-by-side extensibility) offers more possibilities than in-app-extensibility, this tutorial gives you an idea how to build a UI based on a custom business object's OData service in SAP Business Application Studio, test it, and get it into the SAP S/4HANA Cloud system.
-
->Tutorial last updated with SAP S/4HANA Cloud Release 2302
+**Additional Info**
+> - This tutorial gives you an idea how to build a UI with all SAPUI5 options in SAP Business Application Studio, test it, and get it into the SAP S/4HANA Cloud system. If you only require a generated UI and maybe adapting it with restricted options within the SAP S/4HANA Cloud system (in-app-extensibility) check out [Generate the UI for a Custom Business Object and grant Access](abap-extensibility-cbo-ui-generation) and [Adapt the UI of a Business Object](abap-extensibility-cbo-ui-adaptation)
```
Comment (AriannaMussoBarcucci) [inline tutorials/abap-custom-ui-bas-develop-s4hc/abap-custom-ui-bas-develop-s4hc.md:26]: I would argue that if the tutorial shows all the needed steps to build a UI, test it, and get it into the system, you should rephrase this sentence: do not say 'This tutorial gives you an idea' but 'This tutorial shows/illustrates all the needed steps to.. etc' or something similar

**Review by Liebherr — COMMENTED** [review]
> 'before done ' and 'here shown' sounds a bit confusing. What do you mean? That the UI generation was done in a previous part of the tutorial?  

yes, it was done in another tutorial of the same group before

**Review by Liebherr — COMMENTED** [review]
> I would argue that if the tutorial shows all the needed steps to build a UI, test it, and get it into the system, you should rephrase this sentence: do not say 'This tutorial gives you an idea' but 'This tutorial shows/illustrates all the needed steps to.. etc' or something similar 

Well the shown changes are only exemplary and far away from what customers would probably do. it was more about the process then those changes. the idea should only refer to the SAPUI5 options in BAS

**Comment by AriannaMussoBarcucci** [comment]
> > 'before done ' and 'here shown' sounds a bit confusing. What do you mean? That the UI generation was done in a previous part of the tutorial?
> 
> yes, it was done in another tutorial of the same group before

I think the phrasing is confusing, I would re-phrase like 'The ui Generation done in a previous tutorial and the UI adaption shown in this tutorial are so called... etc'

**Review by Liebherr — COMMENTED** [review]
Thank You Arianna, I incorprated both proposals :)

**Pull request merged**